waspc/src/Wasp/Analyzer.hs

```diff
@@ -138,6 +139,7 @@ import qualified Wasp.Psl.Ast.Schema as Psl.Schema
 analyze :: Psl.Schema.Schema -> String -> Either [AnalyzeError] [Decl]
 analyze prismaSchemaAst =
   (left (map ParseError) . parseStatements)
+    >=> (left ((: []) . ValidationError) . validateAst)
```

Comment:
Imagine opening up this PR after reviewing and writing TypeScript all day. And the first thing you see is this line here.

Oh well, time to gaslight myself into thinking Haskell is readable again :smile: